helpers/__tests__/request-test.ts

```diff
@@ -146,6 +147,26 @@ describe('postJSON', () => {
   });
 });
 
+describe('postJSONBody', () => {
+  it('should post without parameters and get json', async () => {
+    const response = mockResponse();
+    window.fetch = jest.fn().mockResolvedValue(response);
+    postJSONBody(url);
+    await new Promise(setImmediate);
+
+    expect(window.fetch).toBeCalledWith(url, expect.objectContaining({ method: 'POST' }));
+    expect(response.json).toBeCalled();
+  });
+
+  it('should post with a body and get json', () => {
+    postJSONBody(url, { data: 'test' });
```

Comment:
Could we test with a bit more complex object to make sure it's correctly transformed ? maybe something like `{ nested: { data: 'test', withArray: [1,2] } }`